### ISSUE TITLE
FR: Use `diff` formatting in Discord webhook codeblocks

### ISSUE THREAD

**Issue by thespad** [tailscale/tailscale#10279]
### What are you trying to do?

Discord webhooks for ACL policy changes are currently very hard to read because they are just big walls of diff text without any formatting.

### How should we solve this?

Use `diff` when creating the code blocks so that Discord formats with the correct colours i.e.

```diff
              "action": "accept",
              "src": [
-                 "autogroup:members",
+                 "autogroup:member",
              ],
```
vs
```
              "action": "accept",
              "src": [
-                 "autogroup:members",
+                 "autogroup:member",
              ],
```

### What is the impact of not solving this?

At the moment it means having to go to the Admin console and look a the audit logs to see a properly highlighted diff for the changes, which partially defeats the point of having the Discord webhooks for audit events.

**Comment by danield555** [comment]
agreed, seeing the diff like we see in tailscale UI itself is enough information and much more readable
